unyt/_array_functions.py

```diff
@@ -938,3 +944,13 @@ def tensordot(a, b, *args, **kwargs):
 def unwrap(p, *args, **kwargs):
     ret_units = p.units
     return np.unwrap._implementation(p.view(np.ndarray), *args, **kwargs) * ret_units
+
+
+@implements(np.interp)
+def interp(x, xp, fp, *args, **kwargs):
+    _validate_units_consistency((x, xp))
+    ret_units = getattr(fp, "units", 1)
```

Comment:
```suggestion
    ret_units = getattr(fp, "units", NULL_UNIT)
```

Reply:
Doesn't really make much difference, the `1` just made me double-take reading this, `NULL_UNIT` makes the logic clearer.

Reply:
I intentionally didn't use `get_units` (which is equivalent to your suggestion) because I wanted the output to match fp's type (so a pure ndarray wouldn't leak as a dimensionless unyt_array). I could add a comment to make that intention clearer, or I could just go with your suggestion and change the test too, no strong opinions actually. Your call !

Reply:
Ah good call please add a comment